web/source/osk/visualKeyboard.ts

```diff
@@ -786,29 +734,6 @@ namespace com.keyman.osk {
           this.highlightKey(key1,true);
         }
       }
-
-      if(key0 && key1 && (key1 != key0) && (key1.id != '')) {
-        //  Display the touch-hold keys (after a pause)
-        this.touchHold(key1);
```

Comment:
Relocated to within the `updateGestures` method.  This does mean that it occurs far earlier (before other base-key processing, rather than after), but it turns out that this doesn't affect functionality or the UX.  Made sure to thoroughly test that during development.